common/scripts/mesh-helper.sh

```diff
@@ -43,21 +43,34 @@ EOF
   fi
 }
 
-generate_br_lan_ip() {
+generate_bridge_ip() {
```

Comment:
Please do not remove the understanding that this function is only for br-lan as there will be quite a few other bridges :)

Reply:
I generalized the name, because the bridge name is configurable.

Reply:
Then please call it multiradiobridge for example, not just "bridge" as that introduces confusion with other bridges :)

Reply:
It can be used also with one radio, that is the reason to generalize it to bridge.

Reply:
Fair enough but I would still call it something clear like LAN_BRIDGE_NAME then for example, especially if it is something coming all the way from mesh.conf and/or NATS server, where I for one would be definitely confused by a mysterious variable only called BRIDGE 🙂